server/src/main/java/io/crate/metadata/table/Operation.java

```diff
@@ -66,6 +67,8 @@ public enum Operation {
         ALTER_BLOCKS, ALTER_OPEN, ALTER_CLOSE, ALTER_REROUTE, REFRESH, SHOW_CREATE, OPTIMIZE);
     public static final EnumSet<Operation> LOGICAL_REPLICATED = EnumSet.of(
         READ, ALTER_BLOCKS, ALTER_REROUTE, OPTIMIZE, REFRESH, COPY_TO, SHOW_CREATE, ALTER_OPEN);
+    public static final EnumSet<Operation> PUBLISHED_IN_LOGICAL_REPLICATION_OPERATIONS = EnumSet.of(
+        READ, UPDATE, INSERT, DELETE, ALTER, ALTER_BLOCKS, ALTER_OPEN, ALTER_CLOSE, ALTER_REROUTE, REFRESH, SHOW_CREATE, COPY_TO, OPTIMIZE);
```

Comment:
it's still draft but just curious -   `RESTORE_SNAPSHOT` and `CREATE_SNAPSHOT` would be also restricted for published tables?